### ISSUE TITLE
Cannot create inspector for DB2 v11.0 on z/OS cause missing "CURRENT ISOLATION" register

### ISSUE THREAD

**Issue by debrando** [ibmdb/python-ibmdbsa#51]
After creating the engine, when instantiating the Inspector:

    reflection.Inspector.from_engine(engine)

...it fails with error:

    [IBM][CLI driver][DB2] SQL0206N "CURRENT" non è valido nel contesto in cui viene utilizzato. SQLSTATE=42703 SQLCODE=-206

The query that fails, issued by the inspector to get the isolation level, is:

    SELECT CURRENT ISOLATION FROM sysibm.sysdummy1

After some research I found that DB2 for z/OS [doesn't have that register]( https://www.ibm.com/support/knowledgecenter/en/SSEPEK_11.0.0/sqlref/src/tpc/db2z_specialregistersintro.html).

**Comment by mphillipson** [comment]
I am receiving the same error attempting to connect to a DB2 z/OS server:
`sqlalchemy.exc.ProgrammingError: (ibm_db_dbi.ProgrammingError) ibm_db_dbi::ProgrammingError: SQLNumResultCols failed: [IBM][CLI Driver][DB2] SQL0206N  "CURRENT" is not valid in the context where it is used.  SQLSTATE=42703 SQLCODE=-206 (Background on this error at: http://sqlalche.me/e/f405)`
We would really like to see this issue addressed if at all possible.

**Comment by abhi7436** [comment]
Hi,
I am looking into this issue. Will update shortly.

Thanks,
Abhinav

**Comment by mixini** [comment]
sqlalchemy.exc.ProgrammingError: (ibm_db_dbi.ProgrammingError) ibm_db_dbi::ProgrammingError: Token ISOLATION was not valid. Valid tokens: DATE TIME TIMESTAMP SERVER TIMEZONE DEGREE. SQLSTATE=42601 SQLCODE=-104 (Background on this error at: http://sqlalche.me/e/f405)

Any update on this? This produces the above error for DB2 for i, upon attempting to initialize the connection.

**Comment by abhi7436** [comment]
Hi,
Apologies for the delay, I am working on the fix. It is taking more time then I expected.

**Comment by Marvin-Schoen** [comment]
Not sure if this helps, but although I get the same error as above when trying to connect via cell magic (% notation), I can connect just fine using pandas: 

```
import pandas as pd
from sqlalchemy.engine import create_engine

engine = create_engine("db2+ibm_db://"+user+":"+passw+"@xxx.xxx.xx:55555/XXX")
#Read Data query into a DataFrame
df = pd.read_sql("""select * from XXX.XXXXX limit 1""", engine)
df.head()
```
I would prefer to use cell magic though.

**Comment by abhi7436** [comment]
Hi All,
I have made the code changes and pushed it into my repository.
These code changes have dependencies in python ibm_db driver. 
It will take us some time to release new version of ibm_db driver.
Once newer version of ibm_db is released I will merge ibm_db_sa code changes to this repository with detailed testing and release it. I have pasted below the change set of my repository and python ibm_db repository, so that user can compile it themselves.

ibm_db_sa:
[77ccc5e](https://github.com/abhi7436/python-ibmdbsa/commit/77ccc5e02481ce197dbd327e8a27844ce093c3ed)

Ibm_db:
[e579f0e](https://github.com/ibmdb/python-ibmdb/commit/e579f0e4ddc473805e220bcaecd1fa1d63389d05)

**Comment by Marvin-Schoen** [comment]
Hi, thanks for your changes. I installed both ibm_db_sa and ibm_da from source including your changes.
Unfortunately another Error occurs:
`name 'SQL_TXN_SERIALIZABLE' is not defined`
which, when looking at [77ccc5e](https://github.com/abhi7436/python-ibmdbsa/commit/77ccc5e02481ce197dbd327e8a27844ce093c3ed) makes sense, as the name is never introduced. I tried introducing it myself, changing the ibm_db.py. I had to guess a feasible value and found that 8 worked just fine (3 didn't). 
So now I can use SQL Alchemy with Jupyter Cell Magic and DB2.
